src/sonarr.ts

```diff
@@ -154,14 +324,40 @@ async function setEpisodeUnmonitored(
 
   if (unmonitorError) {
     throw {
-      message: `Failed to unmonitor ${episodeString}\n${unmonitorError}`,
+      message: `Failed to unmonitor ${episodeString}\n${unmonitorError}.`,
       level: 'error',
     };
   }
 
   console.log(`${episodeString} unmonitored!`);
 }
 
+async function unmonitorSeries(series: Series): Promise<void> {
+  if (!series.ended) {
+    throw {
+      message: `${series.title} has not ended, skipping series unmonitor.`,
+      level: 'warn',
+    };
+  }
+
+  await checkSeasonsWatched(series);
+
+  await setSeriesUnmonitored(series);
+}
+
+async function unmonitorSeason(
+  series: Series,
+  episode: Episode,
+): Promise<void> {
+  const seasonString = `${series.title} - Season ${episode.seasonNumber}`;
+
+  const foundSeason = await checkEpisodesWatched(series, episode, seasonString);
+
+  await setSeasonUnmonitored(series, foundSeason, seasonString);
```

Comment:
Seems like `seasonString` doesn't need to be passed to these functions? It can just be computed from these functions since they have the series title and season number from other parameters? If anything, just create a helper to create the season string and use it within those functions if necessary.